datahub-web-react/src/app/entity/shared/tabs/Dataset/Schema/SchemaTable.tsx

```diff
@@ -161,6 +161,7 @@ export default function SchemaTable({
         dataIndex: 'fieldPath',
         key: 'usage',
         render: usageStatsRenderer,
+        sorter: (sourceA, sourceB) => sourceA.name.localeCompare(sourceB.name),
```

Comment:
I believe we would want to sort this by the count of the usage column and not the name. I'm actually not even sure what `name` would be in this situation here?

Reply:
yeah when I try to use this code to sort, I get a runtime error and react breaks. were you able to find any data locally with usage to test this on?

Reply:
I believe you're going to have to write a function that takes into account `usageStats` above and finds the fields you're sorting and then sorts by `count` that you find by field in `usageStats` above.